pages/intermediate-web-lit.jsx

```diff
@@ -0,0 +1,136 @@
+var React = require('react');
+var Link = require('react-router').Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+
+var Intro = (
+  <div>
+    <h1>Intermediate Web Literacy 1: Intro to CSS</h1>
+    <section className="intro">
+      <Illustration
+      height={204} width={204}
+      src1x="/img/pages/intermediate-web-lit/blocks.png"
+      alt="Multi-colored stacked blocks">
+        <h2>Learn to style HTML elements on a webpage using CSS selectors, attributes, and values.</h2>
+      </Illustration>
+    </section>
+  </div>
+);
+
+var IntermediateWebLit = (
+  <section className="row web-lit-basics">
+    <div className="col-sm-12">
+      <h2>Learning Objectives</h2>
+      <p>
+        Learners will understand and apply coding/scripting, composing, and design with CSS.
+      </p>
+    </div>
+  </section>
+);
+
+var curriculumList = [
+  {
+    title: "Writing the Web",
+    activities: [
+      {
+        title: "CSS Word Pyramid",
+        image1x: "/img/pages/intermediate-web-lit/blue-pyramid.png",
+        originalImgSrc: "http://chadsansing.github.io/curriculum-testing/intermediate-web-lit-two/blue-pyramid.png",
+        caption: "CC0, public domain by ClkerFreeVectorImages",
+        subtitle: "Coding/Scripting, Composing, Design",
+        description: "Learners will use CSS to change the size of text on an HTML webpage, learning about coding, composing, and design. ",
```

Comment:
@hannahkane we might want to wrap keywords with `<strong></strong>` so they appear **bold** like what Chad suggested in the [Google Doc](https://docs.google.com/document/d/1j6kTdhfTION7_N63Ws-qv1NC-V2a0Iuh-4PDkGfQRIE/edit#)

ie. 
```
description: "Learners will use CSS to change the size of text on an HTML webpage, learning about <strong>coding</strong>, <strong>composing</strong>, and <strong>design</strong>.",

```